CommandLineArgsHelper/CommandLineArgsHelper.cs

```diff
@@ -23,7 +23,7 @@ namespace CommandLineArgsHelpers
     public class CommandLineArgsHelper
     {
         // Variables
-        private static readonly Regex _splitterRegex = new Regex(@"^-{1,2}|^\/|=|:", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex _splitterRegex = new Regex(@"^-{1,2}|^\/|=|:|\s+""", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
The fix looks good and simple enough 👍 Thanks for the effort in investigating and contributing the fix! 🙏 

I’ll pull it down and run some tests, and push a new version…should be able to get to it tomorrow.

Reply:
I spoke too soon, after further investigation the robust solution to handle the use cases you've proposed is not this trivial.... see more information on the github issue.